pixi/src/ui/report/CoreWebVitalsReportView.js

```diff
@@ -20,73 +20,52 @@ const CATEGORIES = {
   }
 
   render(data, unit) {
-    this.bar.style.transform = `scale3d(${data.score}, 1, 1)`;
-
-    this.value.textContent = `${(data.numericValue / unit.conversion).toFixed(
+    this.pitch.style.left = `${
+      (data.numericValue / data.proportion.slow) * 100
+    }%`;
+    this.pitch.textContent = `${(data.numericValue / unit.conversion).toFixed(
       1
-    )}${unit.name}`;
-    this.indicator.style.marginLeft = `${data.score * 100}%`;
-    this.value.style.width = `0`;
+    )} ${unit.name}`;
+
+    for (const bar of this.bars) {
+      const label = bar.querySelector('span');
+      const type = bar.getAttribute('data-type');
+      const perc = Math.round(
+        (data.proportion[type] / data.proportion.slow) * 100
+      );
+
+      bar.style.width = `${perc}%`;
+      label.textContent = `${
+        data.proportion[type] / unit.conversion.toFixed(1)
+      } ${unit.name}`;
+    }
   }
 }
 
-class WeightedScale {
+class SimpleScale {
   constructor(container) {
-    this.bar = {
-      positive: container.querySelector('.ap-a-pixi-scale-bar-positive'),
-      negative: container.querySelector('.ap-a-pixi-scale-bar-negative'),
-    };
-    this.pitch = {
-      positive: container.querySelector('.ap-a-pixi-scale-pitch-line-positive'),
-      negative: container.querySelector('.ap-a-pixi-scale-pitch-line-negative'),
-    };
-    this.indicator = container.querySelector('.ap-a-pixi-scale-indicator');
-    this.value = container.querySelector('.ap-a-pixi-scale-value');
+    this.container = container;
+    this.bar = container.querySelector('.ap-a-pixi-scale-chart-bar');
+    this.label = this.bar.querySelector('span');
+    this.pitch = container.querySelector('aside');
   }
 
-  render(data, unit) {
-    // Update bar to match distributions
-    this.bar.positive.style.transform = `scale3d(${data.distributions[0].proportion}, 1, 1)`;
-    this.bar.negative.style.transform = `scale3d(${data.distributions[2].proportion}, 1, 1)`;
-
-    // Update pitch lines
-    this.pitch.positive.style.transform = `translateX(${
-      data.distributions[0].proportion * 100
-    }%)`;
-    this.pitch.positive.textContent = `${
-      data.distributions[0].max / unit.conversion
-    }${unit.name}`;
-
-    this.pitch.negative.style.transform = `translateX(${
-      100 - data.distributions[2].proportion * 100
-    }%)`;
-    this.pitch.negative.textContent = `${
-      data.distributions[2].min / unit.conversion
-    }${unit.name}`;
-
-    // Add a value to the scale and position in distributions
-    this.value.textContent = `${data.percentile / unit.conversion}${unit.name}`;
-    let distributionOffset = 0;
-    for (const distribution of data.distributions) {
-      if (data.percentile < distribution.max) {
-        this.value.style.width = `${
-          (data.percentile / distribution.max) * 100
-        }%`;
-        break;
-      }
+  render(data) {
+    const percentile = Math.round(data.score * 100);
```

Comment:
We still need to keep the distribution from the "good" proportion and set that to the green bar width. We should not be using the score to plot the visual, only providing it in the "Your score" text portion.